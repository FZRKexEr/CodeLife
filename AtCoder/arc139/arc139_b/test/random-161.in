100
54 3 4 2 5 9
73 15 20 11 1 14
87 18 17 17 19 5
43 12 19 4 3 19
92 16 8 2 18 20
40 16 1 19 11 14
7 6 5 5 10 7
51 7 20 16 7 19
50 7 16 2 12 10
8 17 7 1 1 17
8 14 14 8 15 8
39 4 11 5 16 14
4 9 7 15 15 8
20 15 7 6 17 10
32 20 10 11 1 16
21 7 18 13 16 13
33 13 1 3 5 6
47 18 1 20 18 4
30 18 14 10 13 13
42 11 3 13 18 9
60 9 13 8 12 6
43 15 18 2 2 3
16 10 16 11 19 19
70 12 1 10 16 5
13 19 5 2 8 18
34 14 5 13 16 10
13 13 14 17 19 3
37 9 7 8 7 15
1 7 19 1 13 5
47 13 2 3 8 11
93 4 20 2 10 9
63 16 15 9 2 12
94 17 1 11 20 12
93 9 14 20 3 12
27 9 12 15 1 7
40 8 4 1 13 3
51 15 4 12 3 12
4 13 4 20 12 17
86 7 13 11 1 14
72 19 20 8 9 14
55 16 17 7 6 16
100 1 3 20 17 19
23 20 5 4 15 6
44 19 15 10 16 7
34 1 7 12 3 2
82 19 3 3 10 12
27 7 11 5 3 7
12 5 7 1 2 2
17 3 19 19 6 11
84 3 15 4 15 17
20 13 10 6 8 14
98 18 12 6 10 11
62 19 19 20 5 9
47 5 19 18 17 10
88 9 17 10 17 2
82 16 1 20 16 5
82 2 1 4 11 2
85 17 19 16 16 18
23 17 2 12 9 18
9 11 12 4 3 12
45 12 8 11 17 4
28 3 10 10 10 16
57 6 1 6 11 20
26 3 16 6 1 9
54 15 18 20 6 18
15 5 12 6 13 18
71 9 4 12 1 17
24 3 12 17 9 14
7 8 13 6 7 16
96 14 3 4 15 15
87 13 6 15 6 18
33 16 19 11 15 12
76 20 3 10 10 2
41 3 12 1 16 5
59 15 1 6 15 7
18 14 14 5 19 8
16 6 19 12 17 18
18 17 14 14 6 12
41 13 10 15 19 10
39 19 15 11 4 17
34 3 7 1 19 3
78 9 4 3 2 18
95 4 15 12 9 16
88 6 7 19 17 9
6 6 6 14 7 19
63 6 16 1 3 3
33 17 19 16 16 14
60 16 12 14 10 4
33 11 3 13 10 19
46 3 6 13 5 3
95 18 12 9 12 17
43 17 13 5 5 4
9 11 19 7 19 18
15 11 15 19 12 19
14 20 17 17 11 16
100 20 3 20 6 13
3 2 4 14 7 20
29 12 3 13 14 12
9 1 5 19 10 10
44 3 5 15 10 3
