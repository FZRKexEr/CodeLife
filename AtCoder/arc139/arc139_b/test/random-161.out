90
63
53
25
114
134
15
81
34
8
64
68
60
47
39
48
31
80
46
81
66
20
19
130
26
55
19
36
1
48
62
18
199
75
3
30
45
17
34
138
67
644
27
92
26
58
23
4
68
121
28
74
115
109
40
90
164
80
63
36
44
72
117
18
54
36
41
28
42
138
99
41
44
20
129
28
82
20
55
19
18
34
117
141
7
18
238
20
30
39
105
13
63
19
238
30
21
66
50
49
